backend/app/src/main/java/com/moh/phlat/backend/repository/SourceDataFilterSpecification.java

```diff
@@ -0,0 +1,16 @@
+package com.moh.phlat.backend.repository;
+
+import java.util.List;
+
+import org.springframework.data.jpa.domain.Specification;
+
+import com.moh.phlat.backend.model.ProcessData;
+import com.moh.phlat.backend.model.SourceData;
+
+public interface SourceDataFilterSpecification {
+	
+	public Specification<SourceData> buildSpecificationWhereEqual(String columnKey, String value);
+	public Specification<SourceData> buildSpecificationAnd(Specification<SourceData> spec, String columnKey, String value);
```

Comment:
is this getting implemented anywhere?

Reply:
I think I was trying to keep some of the methods as close as possible between Source and Process.  This one has been removed.